plugin/src/main/java/org/opensearch/ml/autoredeploy/MLModelAutoReDeployer.java

```diff
@@ -9,11 +9,7 @@
 
 import static org.opensearch.ml.common.CommonValue.ML_MODEL_INDEX;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Optional;
-import java.util.Queue;
+import java.util.*;
```

Comment:
* --> if you need to setup in your end so that it doesn't create make * automatically.
```To avoid IntelliJ IDEA replacing imports with *, you need to configure the Class count to use import with ‘*’ and Names count to use static import with ‘*’ preferences:

    Go to Preferences > Editor > Code Style > Java.
    Set a big value for both preferences, such as 500.
    Save.
```

Reply:
Thanks for guiding. Fixed.